components/atom/input/src/Input/Component/index.scss

```diff
@@ -18,6 +18,12 @@ $class-read-only: '#{$base-class-input}--readOnly';
     display: none;
   }
 
+  &--readOnly {
```

Comment:
always use native selector over classes
https://www.w3schools.com/cssref/sel_read-only.asp

Reply:
I know its not [yours](https://github.com/SUI-Components/sui-components/commit/a1d6a0e39979bd8e958fd6afea2f80636c2b2e75) but can you remove that class and handle that using the `:read-only` pseudo-selector?

Reply:
+1 to this

Reply:
But the :read-only pseudo-selector is not supported in IE11. We could control it with the HTML attribute or with the class, as it is now (in fact, we have used it because it was already created [here](https://github.com/SUI-Components/sui-components/blob/master/components/atom/input/src/Input/Component/index.scss#L2))